### ISSUE TITLE
Image carousel should have controls to pause autorotation

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#2106]
The carousel on some model pages should have controls to pause auto-rotation.

See https://www.w3.org/WAI/ARIA/apg/patterns/carousel/

Things to check and fix if not working:

* [x] Buttons for displaying the previous and next slides.
* [x] Controls for choosing a specific slide to display
* [x] A button for stopping and restarting rotation
* [ ] Stops rotating when keyboard focus enters the carousel. It does not restart unless the user explicitly requests it to do so.
* [x] Stops rotating whenever the mouse is hovering over the carousel.
* [x] Rotation control is the first element in the Tab sequence inside the carousel
* [x] A carousel container element that encompasses all components of the carousel, including both carousel controls and slides, has either role [region](https://w3c.github.io/aria/#region) or role [group.](https://w3c.github.io/aria/#group)
* [x] The carousel container has the [aria-roledescription](https://w3c.github.io/aria/#aria-roledescription) property set to carousel.
* [x] Each slide container has role [group](https://w3c.github.io/aria/#group) with the property [aria-roledescription](https://w3c.github.io/aria/#aria-roledescription) set to slide.
* [x] Each slide has an accessible label is provided by the property [aria-label](https://w3c.github.io/aria/#aria-label) set on the slide container. If unique names that identify the slide content are not available, a number and set size can serve as a meaningful alternative, e.g., "3 of 10"
* [x] an element wrapping the set of slide elements has [aria-atomic](https://w3c.github.io/aria/#aria-atomic) set to false and [aria-live](https://w3c.github.io/aria/#aria-live) set to:
  * [x] "off" if the carousel is automatically rotating;
  * [x] "polite" if the carousel is NOT automatically rotating.
* [x] The set of slide picker controls is contained in an element with role [group](https://w3c.github.io/aria/#group).
* [x] The group containing the picker controls has an accessible label provided by the value of [aria-label](https://w3c.github.io/aria/#aria-label) that identifies the purpose of the controls, e.g., "Choose slide to display."
* [x] Each picker control is a native button element (recommended) or implements the [button pattern.](https://www.w3.org/WAI/ARIA/apg/patterns/button/)
* [x] The accessible name of each picker button matches the name of the slide it displays.
* [x] The picker button representing the currently displayed slide has the property [aria-disabled](https://w3c.github.io/aria/#aria-disabled) set to true.
